src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+html {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  height: 60px;
+  position: relative;
```

Comment:
Avoid using `position: relative;` if not needed

Reply:
Actually, the only place where you should put is `is-active`, by doing so we allow `::after` to be implemented properly